.esh/commands/update-locales.py

```
@@ -1545,6 +1572,189 @@ def generate_ts_type_recursive(data, indent_level):
 		print(f"  - Error: Failed to write to {output_path}: {e}")
 
 
+
+# --- 主逻辑辅助函数 ---
+def load_single_locale_file(filepath):
+	"""Loads a single locale file and returns its data."""
+	lang = get_lang_from_filename(filepath)
+	if not lang:
+		print(f"  - 警告: 无法从 '{os.path.basename(filepath)}' 提取语言代码，跳过。")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)